src/app/components/Fleet/FleetDetails.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { Breadcrumb, BreadcrumbItem, PageSection, Title } from '@patternfly/react-core';
+import { Fleet } from '@types';
+
+import { useSingleFetch } from '@app/hooks/useSingleFetch';
+
+import FleetDetailsContent from './FleetDetailsContent';
+
+const FleetDetails = () => {
+  // TODO use named parameter in Route
+  const location = useLocation();
+  const urlTokens = location.pathname.split('/');
+  const fleetKey = urlTokens[urlTokens.length - 1];
+
+  const [fleetDetails, hasError, isLoading] = useSingleFetch<Required<Fleet>>(`fleets/${fleetKey}`);
```

Comment:
the order is incorrect
```suggestion
  const [fleetDetails, isLoading, hasError] = useSingleFetch<Required<Fleet>>(`fleets/${fleetKey}`);
```

Reply:
Thanks, I missed it